### DISCUSSION TITLE
Can anyone tell me how to fix this?

### DISCUSSION THREAD

**Discussion by ninjarogue** [LazyVim/LazyVim#1799] [General]
Every time the cursor is placed within a function block, or square brackets (e.g., an object literal) this happens:
<img width="1908" alt="Screen Shot 2023-10-17 at 1 15 50 PM" src="https://github.com/LazyVim/LazyVim/assets/74705062/83c91397-1e07-4d80-ade8-331dcf2c6f7c">
Do you see that strange overlap above the cursor? It's blocking my view and it doesn't go away until the cursor is placed outside of the block. I'm not sure if this is a bug. Is this normal?

Here are the plugins that I've installed: 
- sindrets diffview
- neogit
- some lazyvim plugin extras (typescript, json, leap, codeium, prettier, docker, eslint).

**Comment by folke** [comment]
https://github.com/nvim-treesitter/nvim-treesitter-context

**Comment by ogios** [comment]
```lua
  {
    "nvim-treesitter/nvim-treesitter-context",
    opts = function(_, opts)
      opts.multiline_threshold = 6
    end,
  },
```

**Comment by ninjarogue** [comment]
Would it be possible to disable it like so?

```lua
return {
  {
    "nvim-treesitter/nvim-treesitter-context",
    enabled = false,
  }
}
```

**Reply by ninjarogue** [reply]
I attempted to disable it but this is not working. I'm guessing it has to do with this plugin being a dependency of another plugin?

**Reply by dpetka2001** [reply]
It works for me exactly as you posted it.

**Reply by ninjarogue** [reply]
Okay I know why that didn't work before.. I was making changes to another config that I thought was my lazyvim config.
